e2e/__tests__/v8Coverage.test.ts

```diff
@@ -26,4 +26,18 @@ onNodeVersions('>=10', () => {
     expect(exitCode).toBe(0);
     expect(stdout).toMatchSnapshot();
   });
+
```

Comment:
if you're up for, you can remove the `onNodeVersions('>=10'` stuff - forgotten cleanup when we dropped node 8. You'll need to update the snapshots as well

Reply:
Done. 👍